packages/app/src/debug/empty/DebugBranchError.cy.tsx

```diff
@@ -0,0 +1,10 @@
+import DebugBranchError from './DebugBranchError.vue'
+
+describe('<DebugBranchError />', () => {
+  it('can mount', () => {
+    cy.mount(<DebugBranchError />)
+    cy.contains('No runs found for your branch')
```

Comment:
Could just reference the i18n bundle in this test so we don't inline a bunch of text:

```ts
import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
...
cy.contains(defaultMessages.debugPage.emptyStates.noRunsFoundForBranch)
cy.contains(defaultMessages.debugPage.emptyStates.noRunsForBranchMessage)
```

Reply:
Yes, added in 28d8f3521ff8a7226481387a742309faade5ba60